cli/tests/integration_tests_lsp.rs

```diff
@@ -66,6 +75,50 @@ fn lsp_startup_shutdown() {
   shutdown(&mut client);
 }
 
+#[test]
+fn lsp_init_tsconfig() {
+  let temp_dir = TempDir::new().expect("could not create temp dir");
+  let mut params: InitializeParams =
```

Comment:
```suggestion
  let mut params: lsp::InitializeParams =
```

And remove the use above for `InitializeParams`.